src/states/SelectAttackState.py

```diff
@@ -152,9 +159,39 @@ def update(self, dt, events):
                                 print(f'{defender} takes {damage} damage')
                                 # RENDER DEFENDER ANIMATION
                                 if self.effectOwner == PlayerType.PLAYER:
-                                    self.enemy.ChangeAnimation("death")
+                                    match self.player.selectedCard.name:
+                                        case "Quick Attack" | "Normal Attack" | "Versatile Range Attack" | "Mid Range Attack":
+                                            if self.player.job == PlayerClass.WARRIOR:
+                                                defender.vfx.play("warrior_light_vfx")
+                                            elif self.player.job == PlayerClass.MAGE:
+                                                defender.vfx.play("mage_light_vfx")
+                                            elif self.player.job == PlayerClass.RANGER:
+                                                defender.vfx.play("ranger_light_vfx")
```

Comment:
you can wrap it into one line of code
`defender.vfx.play(f"{self.player.job.value.lower()}_light_vfx")`